shelley/chain-and-ledger/executable-spec/src/Shelley/Spec/Ledger/BaseTypes.hs

```diff
@@ -174,29 +184,29 @@ data StrictMaybe a
 instance NoUnexpectedThunks a => NoUnexpectedThunks (StrictMaybe a)
 
 instance Functor StrictMaybe where
-  fmap _ SNothing  = SNothing
+  fmap _ SNothing = SNothing
```

Comment:
I do like sometimes aligning on the `=`, but this seems like a small price to pay for consistency.